pkgs/tools/filesystems/ceph/default.nix

```diff
@@ -75,6 +75,26 @@ let
     none = [ ];
   };
 
+  getMeta = description: {
+     homepage = "https://ceph.com/";
+     inherit description;
+     license = with licenses; [ lgpl21 gpl2 bsd3 mit publicDomain ];
+     maintainers = with maintainers; [ adev ak johanot krav ];
+     platforms = [ "x86_64-linux" ];
+   };
+
+  ceph-common = python3Packages.buildPythonPackage rec{
+    pname = "ceph-common";
+    inherit src version;
+
+    sourceRoot = "ceph-${version}/src/python-common";
+
+    checkInputs = [ python3Packages.pytest ];
+    buildInputs = with python3Packages; [ pyyaml six ];
```

Comment:
propagatedBuildInputs or nativeBuildInputs, depending on whether they are needed only during build-time of the wheel, or during runtime.

Reply:
Done. Changed to propagatedBuildInputs. :)